src/mlpack/methods/lsh/lsh_main.cpp

```diff
@@ -48,6 +48,10 @@ PARAM_STRING("reference_file", "File containing the reference dataset.", "r",
     "");
 PARAM_STRING("distances_file", "File to output distances into.", "d", "");
 PARAM_STRING("neighbors_file", "File to output neighbors into.", "n", "");
+PARAM_STRING("truth_file", "File of true neighbors for each query. "
+    "If specified, will compute recall (\% of neighbors found). Run with -v "
```

Comment:
I'd consider calling the option --true_neighbors_file instead. Since we provide short option names (like -t) it's reasonable to have long (but descriptive) long parameter names.